.github/workflows/discord-pr-comments.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         run: |
           echo "TITLE=Comment on #$PR_NUMBER $PR_TITLE" >> $GITHUB_ENV
       - name: Build Embeds JSON
+        if: ${{ github.event.sender.login != 'relativeci[bot]' }}
```

Comment:
This PR will have still posted to discord because GH (annoyingly) uses the workflow from master for this workflow